source/features/collapse-markdown-sections.css

```diff
@@ -0,0 +1,7 @@
+.markdown-body > :is(h1, h2, h3, h4, h5, h6) {
```

Comment:
This style will be applied even if the feature is disabled.

After both of these changes are applied, it can be merged